plugins/secure-mode/src/main/java/ai/djl/serving/plugins/securemode/SecureModeAllowList.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.serving.plugins.securemode;
+
+import java.util.Set;
+
+/** A class for properties that are allowlisted in Secure Mode. */
+final class SecureModeAllowList {
+
+    public static final Set<String> PROPERTIES_ALLOWLIST =
+            Set.of(
+                    "engine",
+                    "job_queue_size",
```

Comment:
Add the following to whitelist:
1. max_idle_time
2. batch_size
3. max_batch_delay

Reply:
Yeah, maybe minWorkers and maxWorkers?